components/applications-service/pkg/storage/postgres/service_groups.go

```diff
@@ -76,6 +76,7 @@ const (
       ,s.application as app_name
       ,s.environment as environment
       ,s.service_group_name_suffix as name_suffix
+			,COUNT(s.disconnected) FILTER (WHERE s.disconnected = 'true') AS disconnected_count
```

Comment:
Is `disconnected` a `text`? If it's bool, we should be able to do `FILTER (WHERE s.disconnected)`

Reply:
I'll give that a go, thanks!